diagrams.net/src/index.js

```diff
@@ -14,6 +14,14 @@ import { SyntaxError, TimeoutError, Worker } from './worker.js'
   const worker = new Worker(browser)
   const server = new http.Server(
     micro.serve(async (req, res) => {
+      // Add a /health route that renders a sample diagram by calling the worker
+      if (req.url === '/health') {
+        let sample = `<mxGraphModel><root><mxCell id="0"/><mxCell id="1" parent="0"/></root></mxGraphModel>`
```

Comment:
I guess it can a `const`?

```suggestion
        const sample = `<mxGraphModel><root><mxCell id="0"/><mxCell id="1" parent="0"/></root></mxGraphModel>`
```